tests/testJointcal.py

```diff
@@ -0,0 +1,239 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import unittest
+import os
+import numpy as np
+import collections
+
+import eups
+
+from lsst.afw import geom, coord, table
+from lsst.meas import astrom
+import lsst.utils
+from lsst.jointcal import jointcal
+
+from lsst.daf import persistence
+
+# We don't want the absolute astrometry to become significantly worse
+# than the single-epoch astrometry (about 0.040").
+absolute_error = 42e-3/206265
+# Set to True for a comparison plot and some diagnostic numbers.
+doPlot = False
+
+
+class FakeRef(object):
+    """A mock data ref object, with minimal functionality."""
+
+    def __init__(self, src, calexp_md, dataId):
+        self.src = src
+        self.calexp_md = calexp_md
+        # info about our "telescope"
+        # self.calexp_md.setDouble("AIRMASS", 1)
+        self.calexp_md.setDouble("MJD", 54321.)
+        # self.calexp_md.setDouble("EXPTIME", 30.)
+        self.calexp_md.setDouble("LST", 53.00914)
+        self.calexp_md.setDouble("RA2000", 53.00914)
+        self.calexp_md.setDouble("DEC2000", -27.43895)
+
+        self.dataId = dataId
+        self.dataId['ccd'] = 1
+        self.dataId['tract'] = 1
+
+    def get(self, name, immediate=True):
+        return getattr(self, name)
+
+    def put(self, value, name):
+        setattr(self, name, value)
+
+    def getButler(self):
+        class NamedThing(object):
+            def __init__(self, name):
+                self.name = name
+
+            def getName(self):
+                return self.name
+
+        return {'camera': NamedThing('monkeySim')}
+
+
+class JointcalTest(lsst.utils.tests.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        """Can only do the eups setup once, so have to do it here only."""
+
+        # NOTE: this may be cheating? How else should I do this?
+        jointcal_reference_dir = os.path.join(os.environ['VALIDATION_DATA_JOINTCAL_DIR'],
```

Comment:
That should work. An alternative is to explicitly pass the andConfig argument to the loader constructor (you have to go to the path in question and load the config file from the file named "andConfig.py").